packages/firebase_vertexai/firebase_vertexai/CHANGELOG.md

```diff
@@ -1,3 +1,3 @@
 ## 0.0.1
```

Comment:
I'd go with `0.1.0`. Using a leading `0.0` has caused some confusion in the past. There are no downsides to starting at `0.1.0`.

Reply:
done